### PR TITLE
[MBL-17180][Student] Sync fixes

### PR DESCRIPTION
**Pull request by hermannakos**
Test plan: Test the sync. Check the background progresses after cancel, restart, etc.

refs: MBL-17180
affects: Student
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by tamaskozmer — APPROVED** [review]

File: libs/pandautils/src/main/java/com/instructure/pandautils/features/offline/sync/FileSync.kt [hunk 1]
```diff
@@ -20,6 +20,7 @@ package com.instructure.pandautils.features.offline.sync
 
 import android.content.Context
 import android.net.Uri
+import android.util.Log
```
[the remark this answers is not in the document]
Comment (tamaskozmer) [reply libs/pandautils/src/main/java/com/instructure/pandautils/features/offline/sync/FileSync.kt:23]: Too slow, mate 😄

**Review by kdeakinstructure — APPROVED** [review]
QA 👍

**Pull request merged**